dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1710,6 +1710,19 @@ def test_fiters_file_list(tmpdir, engine):
     )
     assert len(ddf2) == 0
 
+    # Make sure files list can be read with filters when they don't have the same columns order
+    pd.read_parquet(os.path.join(tmpdir, "part.4.parquet"), engine=engine)[
+        ["y", "x"]
+    ].to_parquet(os.path.join(tmpdir, "part.4.parquet"), engine=engine)
+    ddf3 = dd.read_parquet(
+        str(tmpdir.join("*.parquet")),
+        calculate_divisions=True,
+        engine=engine,
+        filters=[("x", ">", 3)],
+    )
+    assert ddf3.npartitions == 3
+    assert_eq(df[df["x"] > 3], ddf3.compute(), check_index=False)
```

Comment:
We typically don't call `compute()` before passing inputs to `assert_eq` as, when a Dask collection is given to `assert_eq` it will  perform a few additional sanity checks as well as implicitly call `compute()`

```suggestion
    assert_eq(df[df["x"] > 3], ddf3, check_index=False)
```

Reply:
Alright, I just updated it.